Source/WebCore/css/parser/CSSPropertyParserWorkerSafe.cpp

```diff
@@ -55,6 +55,13 @@
 
 namespace WebCore {
 
+namespace CSSPropertyParserHelpersWorkerSafe {
+
+RefPtr<CSSValue> consumeFontFeatureSettings(CSSParserTokenRange&);
+RefPtr<CSSPrimitiveValue> consumeFontStretch(CSSParserTokenRange&);
```

Comment:
Is it possible to use the auto-generated functions now using `"parser-exported": true`?

Reply:
Possible, yes. Easy, and should be added to **this** patch, no. The "worker-safe" functions are a bit of a mess right now.

Reply:
I didn't read your note carefully enough. I will use parser-exported in a future patch.